web/src/main/java/org/hillview/RpcTarget.java

```diff
@@ -98,6 +100,11 @@ public Id getId() {
         return this.objectId;
     }
 
+    /**
```

Comment:
Maybe you can add a longer comment explaining why this is necessary.
No one will guess.